crates/ef-testing/src/evm_sequencer/sequencer/mod.rs

```diff
@@ -197,3 +200,87 @@ pub fn convert_contract_class_v1(class: &CompiledClass) -> Result<ContractClass,
         casm_contract_class,
     )?))
 }
+
+lazy_static! {
+    pub static ref INITIAL_SEQUENCER_STATE: SequencerState = {
+        let mut state = SequencerState::default();
+
+        // On KakarotZero we rely on the Cairo1 helpers class for unavailable syscalls and precompiles.
+        #[cfg(feature = "v0")]
+        let storage = vec![
+            (OWNABLE_OWNER, *KAKAROT_OWNER_ADDRESS.0.key()),
+            (KAKAROT_NATIVE_TOKEN_ADDRESS, *ETH_FEE_TOKEN_ADDRESS.0.key()),
+            (KAKAROT_ACCOUNT_CONTRACT_CLASS_HASH, ACCOUNT_CONTRACT_CLASS_HASH.0),
+            (KAKAROT_CAIRO1_HELPERS_CLASS_HASH, CAIRO1_HELPERS_CLASS_HASH.0),
+            (KAKAROT_BLOCK_GAS_LIMIT, Felt::from(BLOCK_GAS_LIMIT)),
+            (KAKAROT_UNINITIALIZED_ACCOUNT_CLASS_HASH, UNINITIALIZED_ACCOUNT_CLASS_HASH.0),
+        ];
+
+        #[cfg(not(feature = "v0"))]
+        let storage = vec![
+            (OWNABLE_OWNER, *KAKAROT_OWNER_ADDRESS.0.key()),
+            (KAKAROT_NATIVE_TOKEN_ADDRESS, *ETH_FEE_TOKEN_ADDRESS.0.key()),
+            (KAKAROT_ACCOUNT_CONTRACT_CLASS_HASH, ACCOUNT_CONTRACT_CLASS_HASH.0),
+            (KAKAROT_BLOCK_GAS_LIMIT, Felt::from(BLOCK_GAS_LIMIT)),
+            (KAKAROT_UNINITIALIZED_ACCOUNT_CLASS_HASH, UNINITIALIZED_ACCOUNT_CLASS_HASH.0),
+        ];
```

Comment:
for `v0` you could only append the `(KAKAROT_CAIRO1_HELPERS_CLASS_HASH, CAIRO1_HELPERS_CLASS_HASH.0)`

Reply:
yes - I tried that, but then the linter would complain of an unused `mut` in the case of `v1` - thus I preferred to split both. Idk what would be the correct approach here.

Reply:
I see, there is already a `#[allow(unused_mut)` for the signature due to this problem in `to_broadcasted_starknet_transaction`.

IMO adding `unused_mut` + comment explaining why will simplify the code but @tcoratger might have a better sense of idiomatic rust

Reply:
ah good idea!